app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/checkemail/CheckEmailScreen.kt

```diff
@@ -0,0 +1,236 @@
+                    color = MaterialTheme.colorScheme.onSurface,
+                    fontSize = MaterialTheme.typography.bodyMedium.fontSize,
+                    fontWeight = FontWeight.Bold,
+                ),
+            )
+            Text(
+                text = descriptionAnnotatedString,
+                textAlign = TextAlign.Center,
+                modifier = Modifier
+                    .padding(horizontal = 24.dp)
+                    .fillMaxWidth()
+                    .wrapContentHeight(),
+            )
+            Spacer(modifier = Modifier.height(32.dp))
+            BitwardenFilledButton(
+                label = stringResource(id = R.string.open_email_app),
+                onClick = remember(viewModel) {
+                    { viewModel.trySendAction(CheckEmailAction.OpenEmailClick) }
+                },
+                modifier = Modifier
+                    .testTag("OpenEmailApp")
+                    .padding(horizontal = 16.dp)
+                    .fillMaxWidth(),
+            )
+            Spacer(modifier = Modifier.height(32.dp))
+            Column(
+                modifier = Modifier.fillMaxSize(),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                val goBackAnnotatedString = CreateAnnotatedString(
+                    mainText = stringResource(
+                        id = R.string.no_email_go_back_to_edit_your_email_address,
+                    ),
+                    highlightText = stringResource(id = R.string.go_back),
+                )
+                ClickableText(
+                    text = goBackAnnotatedString,
+                    onClick = {
+                        goBackAnnotatedString
+                            .getStringAnnotations("URL", it, it)
+                            .firstOrNull()?.let {
+                                viewModel.trySendAction(CheckEmailAction.CloseClick)
+                            }
+                    },
+                )
+                Spacer(modifier = Modifier.height(32.dp))
+                val logInAnnotatedString = CreateAnnotatedString(
+                    mainText = stringResource(
+                        id = R.string.or_log_in_you_may_already_have_an_account,
+                    ),
+                    highlightText = stringResource(id = R.string.log_in),
+                )
+                ClickableText(
+                    text = logInAnnotatedString,
+                    onClick = {
+                        logInAnnotatedString
+                            .getStringAnnotations("URL", it, it)
+                            .firstOrNull()?.let {
+                                viewModel.trySendAction(CheckEmailAction.LoginClick)
+                            }
+                    },
+                )
+            }
+            Spacer(modifier = Modifier.navigationBarsPadding())
+        }
+    }
+}
+
+@Composable
+private fun CreateAnnotatedString(
```

Comment:
Along with the comment on #3620 for maybe breaking this out to the common function there, the convention is to use camelCase for compose functions which return a value.